src/views/StudentDashboard.js

```diff
@@ -102,24 +101,42 @@ export default function StudentDashboard() {
         console.log("err is ", err);
       });
 
-    // const axios_header = {
+    // fetch(`${BACKEND_URL}/stats/student/${student_username}`, {
+    //   method: "POST",
+    //   body: JSON.stringify({}),
     //   headers: {
-    //     Bearer: localStorage.getItem("student_jwt"),
-    //     // "Content-Type": "application/json;charset=UTF-8",
-    //     // "Access-Control-Allow-Origin": "*",
-    //   },
-    // };
-
-    // axios
-    //   .post(`${BACKEND_URL}/stats/student/${student_username}`, axios_header)
+    //     Bearer: `${localStorage.getItem("student_jwt")}`
+    //   }
+    // })
+    //   .then((res) => res.json())
     //   .then((res) => {
     //     console.log(res);
+    //     // setFullName(res.name);
+    //     // setCollegeName(res.college);
+    //     // setEvalStatus(res.evals);
     //   })
     //   .catch((err) => {
-    //     alert("Server error, Try again");
-    //     console.log(err);
+    //     // alert("Server Error, Please try again");
+    //     console.log("err is ", err);
     //   });
 
+    const config = {
+      headers: {
+        Bearer: `${localStorage.getItem("student_jwt")}`,
+      },
+    };
+
+    axios
+      .post(`${BACKEND_URL}/stats/student/${student_username}`, {}, config)
+      .then((res) => {
+        // console.log(res);
+        setStats(res.data);
+      })
+      .catch((err) => {
+        alert("Server error, Try again");
```

Comment:
I don't feel like alert is good way, it blocks the whole page and ruins the experience. You can try to not display the stuff that depends on the failed backend request and show an error instead. 

For example, if this results into an error, store the error and conditionally render the data to show the error message if ran into one.